docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
@pwilkins I think the only question is whether to delete `settings.configure()` (currently commented out) instead of leaving it commented out with my note. I think we should. I just left it there with the comment so others would understand why I cut out some Sphinx boilerplate.

I don't think anyone is asking for anything to be changed, so if so, please mention it.